### ISSUE TITLE
Why is this code not in the App-BrainProducts repository?

### ISSUE THREAD

**Issue by sappelhoff** [labstreaminglayer/App-BrainAmpSeries#2]
I find it counter-intuitive to host this code in its own repository while other support for other amps by Brain Products are hosted in https://github.com/labstreaminglayer/App-BrainProducts

I think that moving the BrainAmp series over there would make it more discoverable for Brain Products equipment users

**Comment by tstenner** [comment]
Mostly for historical reasons – it was one of the first apps I ported to CMake and set up the CI system for so it got its own subdirectory. If it were for me, the other ones would be split up and ported to CMake, but without the hardware and some time it'd have to be done by "someone else"™

**Comment by dmedine** [comment]
I agree that this makes no sense. I would happily have the other amplifier connectors in their own directory instead of grouped together in a bundle. However, I think it would be easier to move 1 in than 4 out. Also, just to muddy the waters a bit more, I plan on releasing quite soon a new app that incorporates LiveAmp, actiCHamp and an RDA client. This makes pretty much the whole Brain Products directory on LSL obsolete without BrainAmp.
